manifest_staging/charts/gatekeeper/templates/gatekeeper-webhook-server-cert-secret.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  annotations: {{- toYaml .Values.secretAnnotations | trim | nindent 4 }}
```

Comment:
same here re: deletion

Reply:
it is indeed a problem caused by the upgraded version of kustomize

Reply:
using version 4 or above, it can be written in such a format, which can solve the problem of missing annotations @sozercan 
<img width="2104" alt="image" src="https://user-images.githubusercontent.com/18587688/211872641-0dbc3388-2678-44b1-9178-feea6522d241.png">

the generated effect display
<img width="1981" alt="image" src="https://user-images.githubusercontent.com/18587688/211873096-96edd7e7-a9d6-4bee-aec4-e26bd43f6e59.png">


reference link https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/